src/components/overlays/explore/Explore.vue

```diff
@@ -152,28 +177,26 @@ export default defineComponent({
     },
 
     methods: {
-        // This next 'disable' is because of the commented out call to Explore. Remove when coded.
-        // eslint-disable-next-line @typescript-eslint/require-await
         async loadPlacesList(): Promise<void> {
             this.loading = true;
-            // TODO: figure out Explore updates and class instance
-            // const placesResult = await this.$store.state.Explore.retrievePlaces();
-            const placesResult = new Array<PlaceEntry>();
+            const placesResult = await Places.getActiveList();
             this.placesList = placesResult;
```

Comment:
This code keeps the loaded places list in local store (inside the Explore.vue overlay). The alternative is to store the places list in Store. This would enable other UI components to use the Places information. I don't know if this would be useful, though.

Reply:
I see the potential benefit of that, might be something to keep on the list of things to maybe do later.